jupyter/nmm-gui.ipynb

```diff
@@ -940,7 +940,7 @@
     "        if (self.player == PLAYER_1 and self.algorithm1 == 'MM') or (self.player == PLAYER_2 and self.algorithm2 == 'MM'):\n",
     "            bestValue, bestMove = bestMove_MM(self.state, self.player, limit = self.algorithm1Limit if self.player == PLAYER_1 else self.algorithm2Limit, visited = self.playedStates)\n",
     "        elif (self.player == PLAYER_1 and self.algorithm1 == 'AB') or (self.player == PLAYER_2 and self.algorithm2 == 'AB'):\n",
-    "            bestValue, bestMove = bestMove_AB(self.state, self.player, limit = self.algorithm1Limit if self.player == PLAYER_1 else self.algorithm2Limit, visited = self.playedStates)\n",
+    "            bestValue, bestMove = bestMove_AB(self.state, self.player, visited = self.playedStates)\n",
```

Comment:
Vielleicht kannst du das noch in der Dokumentation der Klasse `GameState` beschreiben:
Also wo die Parameter `algorithm1Limit` und `algorithm2Limit` beschrieben am Ende des Stichpunktes sowas einfügen:
```
Das Limit hat nur bei dem Algorithmus Minimax eine Verwendung, da Alpha-Beta-Pruning eine iterative Suchentiefe verwendet;
```

Reply:
Das mit der iterativen Suchentiefe hast du doch gemacht, um mich zu testen xD